download-latest.sh

```diff
@@ -65,6 +65,30 @@ get_os() {
     return 0
 }
 
+# Gets the C lib the system is using by setting the $c_lib variable.
+# Returns 0 in case of success, 1 otherwise.
+get_C_library(){
+  if [ "$os" != 'linux' ] ; then
+    return 0
+  fi
+  if ldd --version | grep -qi eglibc; then
+    c_lib='eglibc'
+    return 0
+  elif ldd --version | grep -qi glibc; then
+    c_lib='glibc'
+    return 0
+  elif ldd --version | grep -qi musl; then
+    c_library_failure_usage
+    return 1
+  elif ldd --version | grep -qi bionic; then
+    c_library_failure_usage
+    return 1
+  else
+    return 1
+  fi
+  return 0
```

Comment:
we will never reach the final `return 0` because of the `else return 1`, so let's replace it by `return 1`

Reply:
```suggestion
  fi
  return 1
```